src/screen/dashboard.rs

```diff
@@ -783,6 +785,15 @@ impl Dashboard {
         self.panes.get_mut(pane).map(|state| (pane, state))
     }
 
+    fn get_focused_mut_with_history(
```

Comment:
Nit: By default we normally name mutable functions `_mut` at the end.
```rust
fn get_focused_with_history_mut
```

Reply:
Makes sense, thanks for the note :)